# DNA extraction and 16S amplicon sequencing protocol 
## Version 1.4.0:  December 10, 2018

***

### 1. Background
  This protocol is derived from a “Systematic improvement of amplicon marker gene methods for increased accuracy in microbiome studies” [doi:10.1038/nbt.3601](https://www.nature.com/articles/nbt.3601). Portions of this protocol have been **copied verbatim** from the manuscript and associated protocol [doi:10.1038/protex.2016.030](https://www.nature.com/protocolexchange/protocols/4831) but adapted to our materials and equipment. For publication purposes, I think it will be sufficient to cite the original published protocol.
<br>
The purpose of this protocol is to improve accuracy, reduce chimeras and errors, and implement improved controls for (cross)contamination and monitoring of run-to-run reproducibility in the lab's 16S sequencing projects.
<br>
The protocol uses two steps of PCR: (1) form the primary amplicon in the fewest number of cycles possible, and (2) add barcodes that are compatible with standard Illumina sequencing primers for demultiplexing. The primers that were designed for the protocol can amplify the following variable regions: V1-V3, V3-V4, V3-V5, V4, V4-V6, and V5-V6 of the 16S ribosomal RNA (rRNA) gene, as well as for variable region V9 of the 18S rRNA gene and the ITS1 and ITS2 regions. Because of the dual-indexing approach, and modular design of the 1˚/2˚ PCR, variable regions can be mixed and matched and new primers could even be swapped in without having to purchase new, and indexed primers. See **Table 8.1** for 1˚ PCR primers and the indices in **Table 8.2**.

***
### 2. DNA Extraction
#### 2.1 Theory
Ideally we want a high-throughput extraction method that is scalable and would allow for direct sequencing and qPCR analysis that has internal controls to ensure accuracy and reproducibility of microbial profiles. We have been evaluating the ZymoBIOMICS 96-well kits in both the column filtration and magnetic capture implementations. This protocol uses the magnetic capture as the extraction can be scaled any where from ~8 samples to 96 without wasting extra resources thanks to the nature of magnetic capture and the individual strip tubes in the lysing rack. This approach gives good linear recovery of the difficult to lyse Eggerthella lenta, down to ~1E4 CFU levels as determined by qPCR. This could be easily replaced with the old Promega-based approach, MoBio PowerSoil, or the column-based variant of this kit. Using a copy of the Run_Template.xlsx found in this protocols folder will help facilitate preparation of the sequencing run sheet and track important details of the experiment. *Note: sample names can not contain “.” or "-". Instead use “_”. THIS IS OPPOSITE TO QIIME!*

#### 2.2 Materials
- [ ] ZymoBIOMICS 96 MagBead DNA Kit (Zymo D4302)
- [ ] ZymoBIOMICS Community Standard (Zymo D6300)
- [ ] In-house human microbiota control (See 16S sequencing standards freezer box)
- [ ] 2mL mag-compatible plate (OmniGenX OX1275-S)
- [ ] TemPlate Semi-Skirt 0.2mL PCR Plate (USA Scientific 1402-9200)
- [ ] Beta Mercaptoethanol (BioRad 1610710)
- [ ] Deep well magnetic capture stand (Sigma Z740158)
- [ ] 96x1200µL Tips (Sartorius 791211)
- [ ] 96x200µL tips (USA Scientific 1120-8710)
- [ ] Multichannel Vacuum manifold (Corning 07200564)
- [ ] 96 well plate vortex (IKA MS 3 360-78005-E1)
- [ ] Biospec Mini-Beadbeater-96 or similar*
<br>*Note: plates will not fit in tissuelyzer or MoBio options!*

#### 2.3 Protocol
***Location:** Biological Safety Cabinet used for level 2 work (ideally new coy room). Remember that samples, especially those from humans, could contain a wide variety of pathogens and should be treated with universal caution regardless of their origin. After extraction, these should have been deactivated and samples should be protected from outside DNA contamination. It could be recommended to spin down the plates first as well.*
- [ ] Treat extraction area with UV ~15 minutes or 0.5% bleach.
- [ ] Add 750 µL beta-mercaptoethanol to 150mL bottle of MagBinding Buffer  *(0.5% (v/v) i.e., 500 µl per 100 ml)*
- [ ] Transfer ~50 mg or half fecal pellet, or 200µL liquid sample into lysing plate.
- [ ] Add 650 µL ZymoBIOMICS lysis solution.
- [ ] Disrupt for 5 minutes in Biospec beadbeater (be sure that it is completely tightened and put a folded up large kimwipe under the lid to keep pressure on the strip lids).
- [ ] Incubate at 65˚C for 10 min (optional for Gram +ve organisms or human samples, akin to EMP protocol).
- [ ] Centrifuge 5 minutes at ~3000g.
- [ ] Transfer 200 µL supernatant to 1 mL deep-well plate.
- [ ] Add 600µL ZymoBIOMICS MagBinding Buffer with BME to each well and mix.
- [ ] Add 25 µL Magnetic Beads to each well and mix.
- [ ] Capture for ~2 minutes or until solution is cleared using magnetic capture stand.
- [ ] Discard supernatant.
- [ ] Add 900µL MagWash1 and mix on vortex for 10 minutes (750rpm), capture and discard supernatant.
- [ ] Add 900µL MagWash2 and mix on vortex for 10 minutes (750rpm), capture and discard supernatant.
- [ ] Add 900µL MagWash2 and mix on vortex for 10 minutes (750rpm), capture and discard supernatant.
- [ ] Using 200µL multichannel, remove any residual ethanol that may be in plate.
- [ ] Using 65˚C heat block for 10 minutes, dry off any residual ethanol (dry bath beside HPLC is currently set up for this).
- [ ] Add 100µL nuclease free H2O and capture again using a spacer to raise plate ~5mm.
- [ ] Transfer 50µL supernatant PCR plate for storage.
- [ ] *Optional: spot check negative controls and real samples with Nanodrop. Successful samples should be >10ng/µL with a 260/280>1.6 and 260/230>1.4.  If the 260/230 is <1.4, quantification is not accurate and Qubit should be applied instead. Yield is not an indicator of evenness of extraction.*

***
### 3. Primary PCR
#### 3.1. Theory
To prevent against over-amplification, the primary PCR is carried out as a quantitative PCR that provides real-time feed back on amplification. The goal is to get mid-to-late amplification to carry forward for indexing. To get optimal amplification, a 10-fold dilution series is created and the optimal dilution is selected. In PCR, a 10-fold dilution corresponds to a delay of 3.3 cycles, thus 4x10-fold dilutions cover a range of ~13 cycles. In practice, for mouse samples extracted with the above approach, 10-fold dilutions seems to captured nicely at 20 cycles of PCR with lower density samples (such as pure culture gavages) being captured at the 1X concentration. For these types I would recommend running only two dilutions. For more complex or mixed samples types, It is recommended to do a full 4x10-fold dilutions. PCR inhibition can be directly visualized here as 10-fold dilutions amplifying similarly (or better) than its undiluted stock. Remember there should be ~3 cycles between dilutions. Inhibition has not been seen with the Zymo approach, but has been seen with the Promega based extraction. A direct cause of inhibition is carryover of ethanol containing wash buffer from column based approaches.

#### 3.2. Materials
- [ ] 384 Plates for qPCR (Biorad #HSP3865)
- [ ] 96 Well Plates (USA Sci #1402-9200)
- [ ] Optically clear Plate Seals (Biorad Microseal ‘B’ #MSB1001)
- [ ] DMSO for PCR (Sigma D8418-50mL)
- [ ] SYBR Green I (Sigma S9430) 10x diluted in DMSO to 1000x
- [ ] KAPA HiFi Hot Start PCR kit (KAPA KK2502)
- [ ] Amplification primers of choice at 100µM (see Table 8.1)
- [ ] Nuclease-free H2O (Life Tech 0977-023)

##### Table 3.2.1. Primary PCR Master Mix

Component	| 1 Rxn (µL) | 420 Rxns (µL) 
----------|------------|--------------
Nuclease-free H2O	| 5.3955 | 2606.31
5x KAPA HiFi Buffer	| 1.8	| 756
10 mM dNTPs	| 0.27 |	113.4
DMSO	| 0.45	| 189
1000x SYBR Green	| 0.0045	| 1.89
100 µM Forward Primer	| 0.045	| 18.9
100 µM Reverse Primer	| 0.045	| 18.9
KAPA HiFi polymerase	| 0.18	| 75.6
**Total**	| **9.0**	| **3780.0**

#### 3.3 Protocol
***Location:** PCR hood or separate room/area from other steps*
- [ ] Treat PCR area with UV light for ~15 minutes.
- [ ] Add 9µL of PCR master mix (prepared as per Table 3.2.1) to all wells of a 384 well plate.
- [ ] Divide 384 plate into 4 quadrants and add 1 µL of template DNA to the first quadrant replicating the plate. This layout is at your discretion.
- [ ] Carry out 3x 10-fold dilutions of each sample into the next 3 quadrants by transferring 1 µL with a multichannel being sure to leave at least 1 no template control. Perhaps by not adding control in the most dilute reaction.
- [ ] Amplify using the BioRad CFX384 according to the parameters of Table 3.3.1.

##### Table 3.3.1. Primary PCR Amplification Parameters
Cycle |	Temperature (˚C)  | Time
------|-------------------|------
Initial Denaturation   |	95	| 5 min
20 cycles:
Denature | 98˚C | 20 sec
Anneal | 55˚C	| 15 sec
Extend | 72˚C | 60 sec
Holding	| 4˚C	Hold | 0 sec

\**20 cycles is a good starting point here. With V4 primers, primer dimers will occur by cycle 25 and will make success of amplification difficult.*

***
### 4. Indexing PCR
#### 4.1 Theory
While the primary PCR has created the amplicons for sequencing, there is no identifying information on the DNA to tell the sample of origin. This is the purpose of the indexing PCR. In the former protocol, a single index/barcode was incorporated onto the reverse primer. Here there are two indices/barcodes on both sides of the amplicon. In this way, fewer barcodes can be used as samples are recognized as the combination of left and right barcodes. For example, in our current set up 24 forward, and 16 reverse give 384 combinations. The same amount of sequence capacity would require 384 distinct (and long, and expensive) reverse primers. There are currently 4 replicate sets of primer aliquots in the -20˚C. Please note your usage and date on the bag.
#### 4.2 Materials
- [ ] 96 Well Plates (USA Sci #1402-9200)
- [ ] DMSO for PCR (Sigma D8418-50mL)
- [ ] KAPA HiFi PCR kit (KAPA KK2502)
- [ ] Indexing Primer plate (Pick 1 of 4 for each plate to be sequenced without overlapping).
- [ ] Nuclease-free H2O (Life Tech 0977-023)

##### Table 4.2.1. Indexing PCR Master Mix (3.3x)

Component	| 1 Rxn (20µL rxn) | 110 Rxns
----------|------------------|----------
5x KAPA HiFi Buffer	| 4 | 440
10 mM dNTPs | 0.6 | 66
DMSO | 1.0 | 110
KAPA HiFi polymerase | 0.4 | 44
**Total**	| **6.0** | **660**

#### 4.3 Protocol
***Location:** Anywhere that is not the PCR hood or where final libraries are being prepared. Due to the limited number of cycles, the main risk of contamination here comes from other sequencing libraries.*
- [ ] Transfer 2 µL of non-plateaued reaction into a new 96 well plate containing 198 µL of H2O to create a 100-fold dilution of the primary PCR. Label this for long-term storage.
- [ ] In a new 96 well plate, add 6µL master mix to every well (Table 4.2.1).
- [ ] To each well, add 4µL of indexing primer from a stock plate being sure replicate the same layout. **SPIN INDEX PLATES BEFORE REMOVING SEAL.**
- [ ] Add 10 µL of 100-fold dilution plate being sure to replicate the same layout.
- [ ] Complete amplification as per the protocol in Table 4.3.1.
- [ ] After amplification, use the Qubit (or Quantit kit) to spot check postive and negative controls. Negative controls should be <2ng/µL and positive controls should be >20ng/µL.

##### Table 4.3.1. Indexing Amplification Parameters
Cycle | Temperature (˚C)	| Time
------|-------------------|------
Initial | Denaturation	95 | 	5 min
10 cycles:
Denature | 98˚C | 20 sec
Anneal | 55˚C | 15 sec
Extend | 72˚C | 60 sec
Holding	| 4˚C	Hold | (0 sec)

***

### 5. Amplicon Pooling and Size Selection
#### 5.1 Theory
While it is absolutely still possible to use PicoGreen pooling of amplicons, we are experimenting with SequalPrep kits. The idea is that the wells of this plate have a positively charged coating such that, under low pH conditions, DNA is bound due to its negative backbone charge. Under neutral to basic conditions the DNA is released. The normalization happens due to saturation of binding and thus it is important that the well be saturated with ~250ng per well. After the protocol, all volumes can be multichanneled into a reservoir and mixed. These must then be concentrated as the concentration will be in the range of 0.2-1ng/µL which while usable, is uncomfortably low. To concentrate samples, the speed vac's in the Hooper can be used. If the user in a rush, they can instead use the Qiagen MinElute columns with a vacuum filter.

#### 5.2 Materials
- [ ] SequalPrep Normalization Plate (Life Tech A10510-01)
- [ ] Speed Vac (Savant SC110a)
- [ ] TempPlate Sealing Foil (USA Scientific 2923-0110)
- [ ] MinElute Gel Extraction Kit (Qiagen 28604)
- [ ] TBE or TAE (BioRad 1610773)
- [ ] Agarose for Molecular Biology (BioRad 1613100)
- [ ] Quick-Load 100bp DNA Ladder (NEB N0467S)
- [ ] SYBR Safe DNA Gel Stain (ThermoFisher S33102)
#### 5.3 Protocol
***Location:** lab bench.*
- [ ] Add 15 µL SequalPrep Normalization Binding Buffer to each well of a 96 well plate
- [ ] Transfer 15 µL of PCR production from indexing PCR into each well of a SequalPrep plate
- [ ] Mix by pipetting and incubate at RT for 1h
- [ ] Discard liquid *remember the DNA is bound to plate
- [ ] Wash with 50 µL SequalPrep Normalization Wash Buffer and pipette up and down to mix
- [ ] Completely remove liquid and discard
- [ ] Tap upside down on kim wipes to remove any residual liquid *using vacuum removal may help here.
- [ ] Add 20 µL SequalPrep Elution Buffer. Seal plate with foil. Mix and centrifuge quickly to pull liquid down.
- [ ] Incubate at RT for 5 minutes.
- [ ] Pool eluted DNA into multichannel reservoir.
- [ ] Transfer eluted DNA 500µL at a time in to multiple 1.5mL microcentrifuge tubes.
- [ ] Transfer to Speed Vac set to medium heat with vacuum set to 20 inHg vacuum (500 mmHg).
- [ ] **Wait ~ 3 hours to dry to completion.**
- [ ] Resuspend in 40 µL of H2O moving across tubes to collect all dried DNA into minimum volume.df
- [ ] Prepare 50mL of 1% agarose (0.5g) in the microwave.
- [ ] Add 5 µL of SYBR Safe stain to the gel and mix before pouring into freshly cleaned (with RNase Away or similar) gel mold. Use the smallest lane width possible.
- [ ] Mix 12µL of recovered DNA with 2µL 6x Qiagen Loading Buffer.
- [ ] Load gel along with 5µL of ladder leaving a space between wells.
- [ ] Run for ~45 minutes at 80V.
- [ ] Using blue light tray, excise the band at 427bp into a empty preweighed tube
- [ ] Weigh mass of gel and add 3 volumes buffer QG.
- [ ] Incubate for 10min @ 50˚C or until gel has dissolved
- [ ] Add 1 volume isopropanol (ex, if gel was 100mg, add 100µL).
- [ ] Add mixture to MinElute column.
- [ ] Wash with 500µL buffer QG.
- [ ] Wash with 750µL buffer PE
- [ ] Spin empty column for 2 min at 10,000 g.
- [ ] Elute in 12 µL buffer EB.
- [ ] Check concentration using both NanoDrop and QuBit. Record these in your in the LibraryNorm section of your RunSheet.

***
### 6. Library QC and Quantification
#### 6.1 Theory
While using the Nanodrop is convienient, it is not accurate for concentrations less than ~10-20ng/µL and lacks the sensitivity required. The QuBit/QuantiT uses a fluorescent dye to specifically quantify dsDNA and is generally much more sensitive and specific than the Nanodrop, and once upon a time was how people normalized sequencing libraries. The most up to date way is via qPCR. This is accomplished by using primers against the Illumina sequencing adapters so in effect, you are only quantifying the portion of DNA that can actually be sequenced for the most sensitive quantification. **Accurate quantification is key for getting the correct cluster density on the sequencer.** Sequencing for more information on the importance of cluster density. This step represents one of the most important steps in your library generation and will directly influence the read depth and quality of your sequencing run. Inaccuracy in this step could completely render your sequencing run useless. *Also while not explicitly required, it is not a bad idea to run a 2% gel and/or a HS DNA bioanalyzer chip to look at size distribution in the library and ensure that size selection is not necessary. If V4 amplicon used, band should be ~427 bp and be free of other bands. If other bands are present, size selection will be required.*
#### 6.2 Materials
- [ ] KAPA Library Quantification Kit for Illumina Platforms (KAPA KK4824) *Standards can be ordered separately as they are limiting reagent of kit (KAPA KK4903)*
- [ ] 384 Plates for qPCR (Biorad #HSP3865)
- [ ] Optically clear Plate Seals (Biorad Microseal ‘B’ #MSB1001)
#### 6.3 Protocol
***Location:** lab bench.*
- [ ] If first time using kit, add the 10X primer premix to the KAPA SYBRFast qPCR Master Mix (2X)
- [ ] Using whatever layout you choose, 6µL to 30 wells of a 384 well plate  ((6 standards x 3 replicates) + (3 dilutions of library x 3 replicates) + 3 NTC))
- [ ] Transfer 4 µL of the 6 standards into triplicate wells.
- [ ] Create a CAREFULL dilution series of your library in Tris-Cl pH=8.5 0.05% Tween20 with the following dilutions being sure to mix between:
	- [ ] 2 µL of library into 198 buffer (100x)
	- [ ]	2 µL of library into 198 buffer (10,000x)
	- [ ]	20 µL into 180 buffer (100,000x)
- [ ] Transfer 4 µL of these in triplicate to appropriate wells of 384 well plate.
- [ ] Seal plate with optically clear cover and centrifuge briefly.
- [ ] Amplify according to the parameters in table 6.3.1 on the BioRad CFX384.
- [ ] After completion of the run, export the data. From the Cq results: plate view, transfer appropriate values to the Run Log sheet:LibraryNorm into the highlighted boxes. Use the average fragment length appropriate to your amplicon (427 for V4).
- [ ] To be successful, the amplification efficiency should be 90-110% and the R2 value should be greater than 0.99. *Sanity check: Your QuBit/Nanodrop calculations should be in the same ballpark as the qPCR result. If these are extremely different something has gone wrong. Requantify and run bioanalyzer chip to ensure proper library size distribution.*
##### Table 6.3.1 Library Quantification qPCR
Cycle	| Temperature (˚C) | Time
------|------------------|------
Initial Denaturation | 95 | 5 min
35 cycles:	
Denature | 95 | 30 sec
Anneal/Extend | 60 | 45 sec
Melt Curve | 65->95 | NA

***

### 7. Sequencing
#### 7.1 Theory
While many platforms can be applied for sequencing of the amplicons, MiSeq is among the most common due to its relatively longer read lengths and accuracy. While the NextSeq will provide many more reads, the cost of running and shorter read lengths make it less desirable for some applications. If ultra high-depth is desired perhaps HiSeq V2 Rapid run 2x250 would be appropriate.
Because we are using a different barcoding structure, it is now also unnecessary to add custom sequencing primers as the built in Illumina primers will suffice. Also, we can now provide a sample sheet such that the sequencer demultiplexes our reads for us. To create a sample sheet, an R script (MakeSampleSheet.R) can be used which will harvest the information in your excel file to create the desired output to upload at sequencing time. It is in this github repository. The sample sheet can always be changed after the fact, however it is important that the reads are set up for at least 250x8x8x250 to ensure both barcodes are sequenced.
#### 7.2. Materials
- [ ] MiSeq Reagent Kit (Version 3, 600 cycles, $1,377.00 each)
- [ ] Buffer HT1 (Included with MiSeq Kit)
- [ ] Incorperation buffer (Included with MiSeq Kit)
#### 7.2. Protocol
- [ ] Thaw MiSeq reagent kit overnight in fridge (or in room temp water bath)
- [ ] Generate a MiSeq sample sheet using MakeSampleSheet.R
- [ ] Prepare fresh 0.2 M NaOH (10 µL NaOH and 90µL H2O)
- [ ] In a 1.5 ml microfuge tube, denature 10 μl of 2 nM library by incubating with freshly diluted 0.2 N NaOH for 5 minutes at room temperature.
- [ ] Add 980 μl of Illumina’s HT1 buffer to bring the sample to 20 pM.
- [ ] Dilute to 8 pM by mixing 400 μl of 20 pM library and 600 μl of Illumina’s HT1 buffer in a clean 1.5 ml microfuge tube.
- [ ] Repeat steps d to f to generate 8 pM PhiX library (Start with 2 µL into 8 µL EB to dilute to 2nM)
- [ ] Remove 150 μl of the 8 pM library and discard. Add 150 μl of 8 pM PhiX to the remaining 850 μl of 8 pM library (15% PhiX spike).
- [ ] ix) Add 600 μl of the 8 pM (15% PhiX) library to the sample well of the MiSeq cartridge and initiate sequencing. 

***

### 8. Appendicies

##### Table 8.1 Primers for primary PCR
**Primer name** | **Marker gene** | **Target region** | **Sequence**
----------------|-----------------|-------------------|---------------
V1_27F_Nextera | 16S rRNA | V1-V3 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGAGAGTTTGATCMTGGCTCAG
V3_534R_Nextera | 16S rRNA | V1-V3 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGATTACCGCGGCTGCTGG
V3_357F_Nextera | 16S rRNA | V3-V4, V3-V5 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCTACGGGAGGCAGCAG
V4_515F_Nextera | 16S rRNA | V4, V4-V6 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGGTGCCAGCMGCCGCGGTAA
V4_806R_Nextera | 16S rRNA | V3-V4, V4 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGGGACTACHVGGGTWTCTAAT
V5F_Nextera  | 16S rRNA | V5-V6 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGRGGATTAGATACCC
V5_926R_Nextera | 16S rRNA | V3-V5 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGCCGTCAATTCMTTTRAGT
V6R_Nextera | 16S rRNA | V5-V6, V4-V6 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGCGACRRCCATGCANCACCT
18S_V9_1391_F_Nextera | 18S rRNA | V9 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGGTACACACCGCCCGTC
18S_V9_EukBr_R_Nextera | 18S rRNA | V9 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGTGATCCTTCTGCAGGTTCACCTAC
ITS1F_Nextera | ITS | ITS1 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCTTGGTCATTTAGAGGAAG*TAA
ITS2_Nextera | ITS | ITS1 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGGCTGCGTTCTTCATCGA*TGC
5.8SR_Nextera | ITS | ITS2 | TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGTCGATGAAGAACGCAGCG
ITS4_Nextera | ITS | ITS2 | GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGTCCTCCGCTTATTGATATGC

##### Table 8.2 Indexing primers
**Primer name** | **Primer sequence** | **Index set origin** | **Index name** | **Index sample sheet** | **Primer name**
----------------|---------------------|----------------------|----------------|------------------------|----------------
F1_MetaIndex | AATGATACGGCGACCACCGAGATCTACACTATAGCCTTCGTCGGCAGCGTC | TruSeq i5 | D501 | TATAGCCT  | F1_MetaIndex
F2_MetaIndex | AATGATACGGCGACCACCGAGATCTACACATAGAGGCTCGTCGGCAGCGTC | TruSeq i5 | D502 | ATAGAGGC  | F2_MetaIndex
F3_MetaIndex | AATGATACGGCGACCACCGAGATCTACACCCTATCCTTCGTCGGCAGCGTC | TruSeq i5 | D503 | CCTATCCT  | F3_MetaIndex
F4_MetaIndex | AATGATACGGCGACCACCGAGATCTACACGGCTCTGATCGTCGGCAGCGTC | TruSeq i5 | D504 | GGCTCTGA  | F4_MetaIndex
F5_MetaIndex | AATGATACGGCGACCACCGAGATCTACACAGGCGAAGTCGTCGGCAGCGTC | TruSeq i5 | D505 | AGGCGAAG  | F5_MetaIndex
F6_MetaIndex | AATGATACGGCGACCACCGAGATCTACACTAATCTTATCGTCGGCAGCGTC | TruSeq i5 | D506 | TAATCTTA  | F6_MetaIndex
F7_MetaIndex | AATGATACGGCGACCACCGAGATCTACACCAGGACGTTCGTCGGCAGCGTC | TruSeq i5 | D507 | CAGGACGT  | F7_MetaIndex
F8_MetaIndex | AATGATACGGCGACCACCGAGATCTACACGTACTGACTCGTCGGCAGCGTC | TruSeq i5 | D508 | GTACTGAC  | F8_MetaIndex
F9_MetaIndex | AATGATACGGCGACCACCGAGATCTACACTGAACCTTTCGTCGGCAGCGTC | TruSeq Amplicon | A501 | TGAACCTT | F9_MetaIndex
F10_MetaIndex | AATGATACGGCGACCACCGAGATCTACACTAGATCGCTCGTCGGCAGCGTC | Nextera i5 | N501 | TAGATCGC | F10_MetaIndex
F11_MetaIndex | AATGATACGGCGACCACCGAGATCTACACCTCTCTATTCGTCGGCAGCGTC | Nextera i5 | N502 | CTCTCTAT | F11_MetaIndex
F12_MetaIndex | AATGATACGGCGACCACCGAGATCTACACTATCCTCTTCGTCGGCAGCGTC | Nextera i5 | N503 | TATCCTCT | F12_MetaIndex
F13_MetaIndex | AATGATACGGCGACCACCGAGATCTACACAGAGTAGATCGTCGGCAGCGTC | Nextera i5 | N504 | AGAGTAGA | F13_MetaIndex
F14_MetaIndex | AATGATACGGCGACCACCGAGATCTACACGTAAGGAGTCGTCGGCAGCGTC | Nextera i5 | N505 | GTAAGGAG | F14_MetaIndex
F15_MetaIndex | AATGATACGGCGACCACCGAGATCTACACACTGCATATCGTCGGCAGCGTC | Nextera i5 | N506 | ACTGCATA | F15_MetaIndex
F16_MetaIndex | AATGATACGGCGACCACCGAGATCTACACAAGGAGTATCGTCGGCAGCGTC | Nextera i5 | N507 | AAGGAGTA | F16_MetaIndex
R13_MetaIndex | CAAGCAGAAGACGGCATACGAGATGTCGTGATGTCTCGTGGGCTCGG | TruSeq Amplicon | A701 | ATCACGAC | R13_MetaIndex
R14_MetaIndex | CAAGCAGAAGACGGCATACGAGATCGAGTAATGTCTCGTGGGCTCGG | TruSeq i7 | D701 | ATTACTCG | R14_MetaIndex
R15_MetaIndex | CAAGCAGAAGACGGCATACGAGATTCTCCGGAGTCTCGTGGGCTCGG | TruSeq i7 | D702 | TCCGGAGA | R15_MetaIndex
R16_MetaIndex | CAAGCAGAAGACGGCATACGAGATAATGAGCGGTCTCGTGGGCTCGG | TruSeq i7 | D703 | CGCTCATT | R16_MetaIndex
R17_MetaIndex | CAAGCAGAAGACGGCATACGAGATGGAATCTCGTCTCGTGGGCTCGG | TruSeq i7 | D704 | GAGATTCC | R17_MetaIndex
R18_MetaIndex | CAAGCAGAAGACGGCATACGAGATTTCTGAATGTCTCGTGGGCTCGG | TruSeq i7 | D705 | ATTCAGAA | R18_MetaIndex
R19_MetaIndex | CAAGCAGAAGACGGCATACGAGATACGAATTCGTCTCGTGGGCTCGG | TruSeq i7 | D706 | GAATTCGT | R19_MetaIndex
R20_MetaIndex | CAAGCAGAAGACGGCATACGAGATAGCTTCAGGTCTCGTGGGCTCGG | TruSeq i7 | D707 | CTGAAGCT | R20_MetaIndex
R21_MetaIndex | CAAGCAGAAGACGGCATACGAGATGCGCATTAGTCTCGTGGGCTCGG | TruSeq i7 | D708 | TAATGCGC | R21_MetaIndex
R22_MetaIndex | CAAGCAGAAGACGGCATACGAGATCATAGCCGGTCTCGTGGGCTCGG | TruSeq i7 | D709 | CGGCTATG | R22_MetaIndex
R23_MetaIndex | CAAGCAGAAGACGGCATACGAGATTTCGCGGAGTCTCGTGGGCTCGG | TruSeq i7 | D710 | TCCGCGAA | R23_MetaIndex
R24_MetaIndex | CAAGCAGAAGACGGCATACGAGATGCGCGAGAGTCTCGTGGGCTCGG | TruSeq i7 | D711 | TCTCGCGC | R24_MetaIndex
R1_MetaIndex | CAAGCAGAAGACGGCATACGAGATTCGCCTTAGTCTCGTGGGCTCGG | Nextera i7 | N701 | TAAGGCGA | R1_MetaIndex
R2_MetaIndex | CAAGCAGAAGACGGCATACGAGATCTAGTACGGTCTCGTGGGCTCGG | Nextera i7 | N702 | CGTACTAG | R2_MetaIndex
R3_MetaIndex | CAAGCAGAAGACGGCATACGAGATTTCTGCCTGTCTCGTGGGCTCGG | Nextera i7 | N703 | AGGCAGAA | R3_MetaIndex
R4_MetaIndex | CAAGCAGAAGACGGCATACGAGATGCTCAGGAGTCTCGTGGGCTCGG | Nextera i7 | N704 | TCCTGAGC | R4_MetaIndex
R5_MetaIndex | CAAGCAGAAGACGGCATACGAGATAGGAGTCCGTCTCGTGGGCTCGG | Nextera i7 | N705 | GGACTCCT | R5_MetaIndex
R6_MetaIndex | CAAGCAGAAGACGGCATACGAGATCATGCCTAGTCTCGTGGGCTCGG | Nextera i7 | N706 | TAGGCATG | R6_MetaIndex
R7_MetaIndex | CAAGCAGAAGACGGCATACGAGATGTAGAGAGGTCTCGTGGGCTCGG | Nextera i7 | N707 | CTCTCTAC | R7_MetaIndex
R8_MetaIndex | CAAGCAGAAGACGGCATACGAGATCCTCTCTGGTCTCGTGGGCTCGG | Nextera i7 | N708 | CAGAGAGG | R8_MetaIndex
R25_MetaIndex | CAAGCAGAAGACGGCATACGAGATCTATCGCTGTCTCGTGGGCTCGG | TruSeq i7 | D712 | AGCGATAG | R25_MetaIndex
R10_MetaIndex | CAAGCAGAAGACGGCATACGAGATCAGCCTCGGTCTCGTGGGCTCGG | Nextera i7 | N710 | CGAGGCTG | R10_MetaIndex
R11_MetaIndex | CAAGCAGAAGACGGCATACGAGATTGCCTCTTGTCTCGTGGGCTCGG | Nextera i7 | N711 | AAGAGGCA | R11_MetaIndex
R12_MetaIndex | CAAGCAGAAGACGGCATACGAGATTCCTCTACGTCTCGTGGGCTCGG | Nextera i7 | N712 | GTAGAGGA | R12_MetaIndex

##### Table 8.3 Expected Amplicon Sizes
**Target Gene** | **Variable Region** | **Size (bp)** 
----------------|---------------------|---------------
16S | V1-V3 | 662 +/- 20%
16S | V3-V4 | 550 +/- 20%
16S | V3-V5 | 722 +/- 20%
16S | V4 | 427 +/- 20%
16S | V4-V6 | 685 +/- 20%
16S | V5-V6 | 416 +/- 20%
18S | V9 | 260 +/- 20%
ITS | NA | HIGHLY VARIABLE



